crates/core/tedge_agent/src/operation_workflows/persist.rs

```diff
@@ -279,16 +275,12 @@ impl WorkflowRepository {
             if let Err(err) = tokio::fs::copy(source.clone(), target.clone()).await {
                 error!("Fail to persist a copy of {source} as {target}: {err}");
             } else {
-                self.in_use_copies.insert(version.clone(), 1);
+                self.in_use_copies.insert(version.to_string(), 1);
```

Comment:
Minor, `to_owned` better communicates the intent here
```suggestion
                self.in_use_copies.insert(version.to_owned(), 1);
```

Reply:
Done: https://github.com/thin-edge/thin-edge.io/pull/3309/commits/eecd22274eb3baed62ffefa53be54d1768dc1c99